Makefile

```diff
@@ -2,38 +2,57 @@ LISP ?= sbcl
 
 all: build
 
+# can use bespoke dir like 'QLDIR=~/nostandard/local-projects make'
+QLDIR ?= $(HOME)/quicklisp/local-projects
```

Comment:
this is not exactly the QL dir but its local-projects directory, I'd change sthg to be more precise.